src/server/server_family.h

```diff
@@ -291,6 +291,9 @@ class ServerFamily {
                        [&match](auto& elem) { return elem.state == match; });
   }
 
+  // Used by DFLY command after replica takeover
```

Comment:
Please also add what it does (aka: updates old master node config by removing itself from the map so it could point to the promoted replica) or smthing like this